src/server/cluster/cluster_slot_migration.cc

```diff
@@ -17,15 +19,29 @@ ABSL_DECLARE_FLAG(int32_t, port);
 namespace dfly {
 
 using namespace std;
+using namespace util;
 using namespace facade;
 using absl::GetFlag;
 
-ClusterSlotMigration::ClusterSlotMigration(string host_ip, uint16_t port,
+namespace {
+// Distribute flow indices over all available threads (shard_set pool size).
+vector<vector<unsigned>> Partition(unsigned num_flows) {
+  vector<vector<unsigned>> partition(shard_set->pool()->size());
+  for (unsigned i = 0; i < num_flows; ++i) {
+    partition[i % partition.size()].push_back(i);
+  }
+  return partition;
+}
+
+}  // namespace
```

Comment:
nit: we can move it to protocol client as we have the same in replication

Reply:
I don't think that protocol_client is the correct place for it. I agree that we need some another place for it, so I will think about it in the future PRs